### ISSUE TITLE
[ANSIBLE0015][UsingBareVariablesIsDeprecatedRule] False positive when multiline jinja template

### ISSUE THREAD

**Issue by gzm55** [ansible/ansible-lint#250]
# Issue Type
- Bug report

# Ansible and Ansible Lint details

```
ansible --version
ansible 2.2.1.0

ansible-lint --version
ansible-lint 3.4.11
```

- ansible installation method: OS package
- ansible-lint installation method: pip

# Desired Behaviour

For multiline template in `with_flattened` and left newline chars in the multi line parsing result, rule ANSIBLE0015 is always not happy:

```pl.yml
---
- hosts: all
  tasks:
  - name: test
    debug:
    with_flattened: >
      {{  'a',
          'b' }}
```

and runing `ansible-lint pl.yml` should be successful.

# Actual Behaviour (Bug report only)

```
[ANSIBLE0015] Found a bare variable '{{  'a',
    'b' }}
' used in a 'with_flattened' loop. You should use the full variable syntax ('{{{{  'a',
    'b' }}
}}')

```
